agetodays-convertor.js

```diff
@@ -1,14 +1,7 @@
-/**
- * @param {number} year is to be raise
- * @param {number} year must be a natural number
- * @function {number} loop finds how many leap years between 
-   given year to current year
- * @var {number} age, it find age
- * @var {number} nonLeapyear,it finds the number of nonLeapyear
- * @return {number} returns days of the given year to current year
-*/
+

```

Comment:
/**
 * convert age to days
 * @param {number} year - Year of Birth
 * @returns {number} - returns your calculated age to days
 */